onboarding/mentors.md

```diff
@@ -31,53 +71,83 @@ It's up to the mentoring pair to find what works best for them and to ensure the
     - Their manager
     - Their team lead
     - Artsy atlas, calendars, benefit websites, mail filters, Jira and Github management tools...
-  
 
-### Your first meeting
+**Your first meeting**
+
 Get to know each other. This can be a nice, casual conversation. Focus on active listening.
+
 - Talk about your own history at and before joining Artsy
 - Where are they coming from?
-  - If they will be remote, where are they from? What is it like there & what works for them in terms of remote work?
+  - If they will be remote, where are they from? What is it like there & what works for them in terms of remote
+    work?
   - Are they new in town? Where do they live, for how long? Just have a normal conversation.
 - Prior experience with our [technology stacks](/playbooks/technology-choices.md) (and others)
 - Other relevant past experience, particularly 'soft skills' that might seem orthogonal to writing code
-- Are they using the [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)? If so, check in on where they are and offer to be a resource on things there. Some tasks might stand out as good things to do in between People Ops meetings, while working nearby each other or to pair on.
+- Are they using the
+  [onboarding checklist](https://github.com/artsy/README/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md)?
```

Comment:
This has been moved to artsy/potential `https://github.com/artsy/potential/blob/master/.github/ISSUE_TEMPLATE/engineering-onboarding.md`.